src/core/Directus/Application/Application.php

```diff
@@ -142,7 +142,7 @@ public function setCheckRequirementsFunction(\Closure $function)
     protected function createConfig(array $appConfig)
     {
         return [
-            'settings' => ArrayUtils::pull($appConfig, 'settings', []),
+            'settings' => $appConfig['settings'],
```

Comment:
May I know why you remove the condition of considering the default value? Currently, It is throwing the error of undefined settings in the API. Kindly revert this change if this is not necessary or add `isset` condition.

Reply:
That call is removing settings from config, which was being used in logger (it never worked to be honest). I'll check another approach.

Also, what files do you mean? Would you mind listing them? Should I just *remove* them?